northstar/src/runtime/token.rs

```diff
@@ -51,21 +48,29 @@ pub(crate) enum VerificationResult {
 /// Token instance
 #[derive(Clone, PartialEq)]
 pub(crate) struct Token {
+    /// Duration how long the token is valid once created
+    validity: time::Duration,
+    /// Creatin time
     time: time::Duration,
```

Comment:
That doesn't make any difference since the age is always compared by durations. If this would be an `Instant` there would be at each comparison a `time.elapsed()` in order to compare it to the max validity.

Reply:
well yes, that would be expected, don't see the problem. Using duration for a time point is a bit weird because the implicit epoch. With `Instant` it is clear what the epoch is.